packages/core/admin/admin/src/translations/en.json

```diff
@@ -489,6 +489,7 @@
   "content-manager.App.schemas.data-loaded": "The schemas have been successfully loaded",
   "content-manager.DynamicTable.relation-loaded": "Relations have been loaded",
   "content-manager.DynamicTable.relation-loading": "Relations are loading",
+  "content-manager.DynamicTable.relation-search-loading": "Entries are loading",
```

Comment:
I was wondering! I used "Relations are loading" for the RelationList but it felt strange to use the same for the search entries since they are not yet relations until the user adds them - wdyt?

Reply:
I don't have a strong opinion, but I feel like they are close enough to be merged. If you keep the split we should use `Entities`, because that is what we call them :)

Reply:
mmh let's go for "Relations are loading" for both, with a `loadingMessage={ id, defaultMessage }` props to RelationInput, might not be worth it to decouple them!